### PR TITLE
Adjustments to the animation control - requires Zeq #98

### PR DESCRIPTION
**Pull request by rdumusc**

### CODE REVIEW

**Comment by jafyvilla** [comment]
+1

**Comment by rdumusc** [comment]
retest this please

File: livre/eq/zeq/communicator.cpp [hunk 1]
```diff
@@ -236,17 +236,21 @@ class Communicator::Impl
...
-        auto frameSettings = _config.getFrameData().getFrameSettings();
-        auto& params = _config.getApplicationParameters();
-
         const Vector2ui& frameRange = _config.getDataFrameRange();
         const uint32_t frameCount = frameRange[1] - frameRange[0];
 
-        // Continue polling until new frames received
-        if( frameCount > 0 &&
-            frameSettings->getFrameNumber() == frame.current  )
+        if( frameCount <= 0 )
```
Comment (bilgili) [inline livre/eq/zeq/communicator.cpp]: Framecount cannot be negative.

**Comment by rdumusc** [comment]
Updated

**Comment by favreau** [comment]
Retest this please

**Pull request merged**